src/model/markercollection.js

```diff
@@ -331,17 +338,25 @@ class Marker {
 		 */
 		this._managedUsingOperations = managedUsingOperations;
 
+		/**
+		 * Flag indicates if the marker changes the data model.
+		 *
+		 * @protected
```

Comment:
Why did you make it protected? It does not need to be used in the tests (although, it is) AFAICS.

Reply:
It's used to check whether the getter reflects its changes. I'll fix that test as for now it's artificial a bit. But I see, that live ranges are used in tests too few times and these tests would be hard to change, so I'll mark that property as protected.